packager/VisibilityChecker.cc

```diff
@@ -4,13 +4,9 @@
 #include "absl/synchronization/blocking_counter.h"
 #include "ast/treemap/treemap.h"
 #include "common/concurrency/ConcurrentQueue.h"
-#include "common/sort/sort.h"
-#include "common/strings/formatting.h"
```

Comment:
ooc, were these includes in service of assorted rbi-gen stuff? or do you think they became unused before we deleted the rbi-gen stuff?

Reply:
Probably before the rbi-gen stuff. For example, [this commit](https://github.com/sorbet/sorbet/commit/c54747bf7abbd0b2bc65bee5fc3b4deb3b4d22ae) got rid of the call to `fast_sort`. I assume something similar for the other `include`s.